src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -57,20 +58,19 @@ const PackageManagerSection = ({ packageManagers, prettyOSName }) => {
          *
          * ref: https://app.asana.com/0/1201010428539925/1201881376116200/f
          */
-        <Tabs showAnchorLine={false}>
+        <CodeTabs tabs={packageManagers.map(({ label }) => label)}>
```

Comment:
GitHub won't let me make a comment on the `TODO`, but that comment can be removed now! 🎉

Reply:
dang it ahah i thought i did this - thanks for catching it - removed! ✅